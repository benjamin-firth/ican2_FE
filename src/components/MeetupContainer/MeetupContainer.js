import React from 'react';
import './MeetupContainer.scss';
import Meetup from '../Meetup/Meetup';

const MeetupContainer = () => {
  return (
    <section className='meetups-container'>
      <Meetup />
      <Meetup />
    </section>
  );
}

export default MeetupContainer;
