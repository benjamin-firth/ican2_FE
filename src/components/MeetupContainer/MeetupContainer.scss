@import '../../variables.scss';

.meetups-container {
  padding-top: 12vw;
  height: auto;
  min-height: 100vw;
  background-color: $primarycolor;
}
