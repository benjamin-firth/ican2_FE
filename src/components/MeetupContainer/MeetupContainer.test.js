import React from 'react';
import { shallow } from 'enzyme';
import MeetupContainer from './MeetupContainer';

describe('MeetupContainer', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow( <MeetupContainer />)
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

});