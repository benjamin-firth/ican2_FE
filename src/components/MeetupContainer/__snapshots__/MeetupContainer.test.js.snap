// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MeetupContainer should match the snapshot 1`] = `
<section
  className="meetups-container"
>
  <Meetup />
  <Meetup />
</section>
`;
